support/overlay/overlay.nix

```diff
@@ -63,6 +63,13 @@ in
 
     gxlimg = callPackage ./gxlimg { };
 
+    meson64-tools = let
+      src = builtins.fetchTarball {
+        url = "https://github.com/angerman/meson64-tools/archive/a2d57d11fd8b4242b903c10dca9d25f7f99d8ff0.tar.gz";
+        sha256 = "1487cr7sv34yry8f0chaj6s2g3736dzq0aqw239ahdy30yg7hb2v";
+      };
+    in (import src { nixpkgs = final; }).meson64-tools;
+
```

Comment:
This is still an IFD; we'll need to package it locally with a proper `callPackage` use.

Reply:
I don't think it actually is. For one, it builds with --no-allow-import-from-derivation, and in addition, fetchTarball doesn't produce a derivation, it produces the result directly (even though it might possibly create a derivation for it internally).

Reply:
Oh... right... It's technically not an IFD, but we still want to package it ourselves.

Please use this, as it's intended to go in Nixpkgs upstream at some point, and eventually we'll drop our packaging of it.

 - https://github.com/NixOS/nixpkgs/blob/f02c05e056b9bef1b8ebc445a3046432a68d4314/pkgs/misc/meson64-tools/default.nix

The tool names in the builder should also use the `meson64-*` qualified aliases this adds. To ensure the generic names are not trampled by other tools.